src/Workspaces/Core/Portable/Options/ILegacyWorkspaceOptionService.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.Options;
+
+internal interface ILegacyWorkspaceOptionService : IWorkspaceService
```

Comment:
is this a temporary thing for us?  can we doc/obsolete this perhaps?

Reply:
Added doc.

Reply:
I think we can add `Obsolete` on `Solution.Options` in follow up PR.